docker-compose.yaml

```diff
@@ -46,7 +46,7 @@ services:
             # uncomment after Step 2 of README
             # - ./local_configs/backend/:/etc/artemis/
             # comment after Step 2 of README
-            - ./backend/supervisor.d/:/etc/supervisor/conf.d/
+            - ./backend/supervisor.d/:/etc/supervisor.d/
```

Comment:
shouldn't the same be done for the monitor?

Reply:
monitor is ubuntu because of bgpstream

Reply:
noting not to forget: check also the instructions on the wiki page about these locations (if changed, they need to be updated after merging).